MachObfuscator/Mach/Mach+Replacing.swift

```diff
@@ -19,6 +31,27 @@ private extension Mach {
         if let classNameSection = objcClassNameSection {
             data.replaceStrings(inRange: classNameSection.range.intRange, withMapping: map.classNames)
         }
+        if let methTypeSection = objcMethTypeSection {
+            // This is very naive and simple algorithm and not efficient at all.
+            // Its main advantage is that it does not require parsing and generating methType strings
+            // Class names seem to be always surrounded by some kind of special characters, like " or < and >
+            data.replaceStrings(inRange: methTypeSection.range.intRange, withMapping: { methType in
```

Comment:
What do you think about extracting this algorithm (together with all the comments) to some other method? I think such large block of code obscures this method a bit.

Reply:
done